.github/workflows/flutter_integration.yaml

```diff
@@ -47,9 +47,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v3
         with:
-          java-version: '8.x'
+          distribution: 'temurin'
```

Comment:
[In AAT](https://github.com/ably/ably-asset-tracking-android/blob/main/.github/workflows/assemble.yml#L16) we are using the `zulu` distribution, why have you chosen `temurin` and not something else? :thinking:

Reply:
Both `Temurin` and `Zulu` are okay - I chose `Temurin` because it follows the footsteps of AdoptOpenJDK - it's open-source and maintained by a few companies under Eclipse. `Zulu` is only maintained by `Azul`. I've followed tips on [whichjdk.com](whichjdk.com)